shared/constants/search.js

```diff
@@ -7,10 +7,10 @@ export type SearchPlatforms = 'Keybase' | 'Twitter' | 'Github' | 'Reddit' | 'Coi
 
 export type ExtraInfo = {
   service: 'external',
-  icon: IconType,
+  icon: ?IconType,
   serviceUsername: string, // i.e. with twitter it would be malgorithms
-  serviceAvatar: ?string, // i.e. with twitter it would be their twitter avatar url
-  fullNameOnService: ?string // Say with twitter we know malgorithms is "Chris Coyne"
+  serviceAvatar: string, // i.e. with twitter it would be their twitter avatar url
```

Comment:
we can use '' instead of null then we can have a simpler type right, its always a string and we can still do the same truthy checks against it